sdk/identity/Azure.Identity/src/InteractiveBrowserCredential.cs

```diff
@@ -89,32 +151,66 @@ public override async ValueTask<AccessToken> GetTokenAsync(TokenRequestContext r
             return await GetTokenImplAsync(true, requestContext, cancellationToken).ConfigureAwait(false);
         }
 
+        private async Task<AuthenticationRecord> AuthenticateImplAsync(bool async, TokenRequestContext requestContext, CancellationToken cancellationToken)
```

Comment:
Nit - we're supposed to shy away from `Impl` and use `Core` so our callstacks look cleaner when an exception is thrown.  Doesn't have to be fixed in this PR.

Reply:
Is that captured in guidelines? We need to be doing a better job of keeping those up to date. I wasn't aware of this change either, but knew archboard wasn't crazy about abbreviations even in internal code, so have been using "...Internal" instead of "...Impl", which I see used quite a bit in .NET or even our own track 2 code.